data/json/itemgroups/Weapons_Mods_Ammo/magazines.json

```diff
@@ -455,6 +455,7 @@
       { "item": "blrmag", "prob": 20, "charges": [ 0, 0 ] },
       { "item": "ruger_pr_mag", "prob": 20, "charges": [ 0, 0 ] },
       { "item": "ruger_arr_mag", "prob": 25, "charges": [ 0, 0 ] },
+      { "item": "steyr_scout_mag", "prob": 25, "charges": [ 0, 0 ] },
```

Comment:
I found 12 steyr scout mags on gunbroker, and 69 p90 mags. I'd move your magazines to the rare rifle magazine groups, and to dertermine your spawn weights I'd change multiply the p90's by (12/69), and round up.